Java/Algorithms/Searching/LinearSearch.java

```diff
@@ -0,0 +1,32 @@
+import java.util.*;
+class LinearSearch
+{
+   public static void main(String args[])
+   {
+      int c, num, item, array[];
+      
+      Scanner sc = new Scanner(System.in);
+      System.out.println("Enter number of elements:");
+      num = sc.nextInt(); 
+      //Creating array to store the all the numbers
+      array = new int[num]; 
+      System.out.println("Enter " + num + " integers");
+      //Loop to store each numbers in array
+      for (c = 0; c < num; c++)
+        array[c] = input.nextInt();
+
+      System.out.println("Enter the search value:");
+      item = sc.nextInt();
+
+      for (c = 0; c < num; c++)
```

Comment:
Create new function for linear search

Reply:
Changes have been updated and committed through same branch. Plz Check and further notify if any update required. Thank You.